rules/windows/process_creation/proc_creation_win_html_help_spawn.yml

```diff
@@ -22,7 +26,27 @@ detection:
             - '\regsvr32.exe'
             - '\wmic.exe'
             - '\rundll32.exe'
-    condition: selection
+            - '\MSHTA.EXE'
+            - '\CertUtil.exe'
+            - '\CertReq.exe'
+            - '\MSbuild.exe'
+            - '\installutil.exe'
+            - '\schtasks.exe'
+            - '\msiexec.exe'
+    selection_2:
+        Image|endswith:
+            - '\Windows\hh.exe'
+            - 'C:\Windows\SysWOW64\hh.exe'
+        CommandLine|contains:
+            - '\Downloads\'
+            - '\Content.Outlook\'
+            - '\AppData\Local\Temp\Temp?_'
+            - '\AppData\Local\Temp\Rar$'
+            - '\AppData\Local\Temp\7z'
+            - '\AppData\Local\Temp\wz'
+            - '\AppData\Local\Temp\peazip-tmp'
```

Comment:
Why not simply '\AppData\Local\Temp\' ?

Reply:
The idea from elastic was to avoid FP I think? since installers and other stuff use temp. So the focus here was on files opened from compressed files such as `.7z`, `.zip`, `.rar`...etc

Reply:
Get '\Downloads\' in the contains, so seem to me more a list of path view in the wild.

Reply:
Get '\Downloads' in the contains, so seem to me more a list of path view in the wild.

Reply:
You're right. We could add it and see :D It won't hurt I guess. Leave it for you to decide.

Reply:
let's start with a `\AppData\Local\Temp\`  and keep the other a comment .

Reply:
Thanks for changing it